### DISCUSSION TITLE
outstanding issues with my GitHub account.

### DISCUSSION THREAD

**Discussion by sajibAdhi** [community/community#51497] [GitHub Education]
![Web capture_30-3-2023_203442_education github com](https://user-images.githubusercontent.com/60180521/228870884-2c0687d9-7733-46b2-8fc2-ba37735c83c3.jpeg)

**Comment by samus-aran** [comment]
Hi @sajibAdhi ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was closed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

[You're original post is here.](https://github.com/orgs/community/discussions/49494) 

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
